src/mmwtex.c

```diff
@@ -30,7 +30,7 @@ flag g_htmlFlag = 0; // HTML flag: 0 = TeX, 1 = HTML
 flag g_altHtmlFlag = 0;
 // Output statement lists only, for statement display in other HTML pages,
 // such as the Proof Explorer home page.
-flag g_briefHtmlFlag = 0;
+flag g_briefHtmlFlag = 0; // TODO: unused
```

Comment:
Unused? What about https://github.com/metamath/metamath-exe/blob/ae35c1b2931be8f9683f8bf66d1d10c9b56db706/src/metamath.c#L2854
And https://github.com/metamath/metamath-exe/blob/ae35c1b2931be8f9683f8bf66d1d10c9b56db706/src/metamath.c#L2616

Reply:
And
https://github.com/metamath/metamath-exe/blob/ae35c1b2931be8f9683f8bf66d1d10c9b56db706/src/metamath.c#L2578-L2587
which shows that it is not a constant either.

Reply:
Should I move the declaration/initialization `flag g_briefHtmlFlag = 0;` to `metamath.c` ?

Reply:
I think it could even be moved to a local variable in `command()`, one less global variable to worry about.

Reply:
done in 6bc25466c16958979ad8ab106381c3049d086606